### ISSUE TITLE
vue/keyword-spacing (and several other wrapper rules) not working for code inside <script></script>

### ISSUE THREAD

**Issue by codynguyen** [vuejs/eslint-plugin-vue#872]
Hi, the `vue/keyword-spacing` rule and other rules that wraps core eslint rules don't seem to lint code inside `<script></script>` in a .vue file. Code inside `<template></template>` does get checked and the rules work.

**Tell us about your environment**	
* **ESLint version:**	5.16.0
* **eslint-plugin-vue version:**	commit [c3111fe](https://github.com/vuejs/eslint-plugin-vue/commit/c3111fed451807eb92dc3f5020d144fbd2af9fda) (latest master code)
* **Node version:**	 8.11.2

**Please show your full configuration:**

```js
module.exports = {
  parserOptions: {
    parser: "babel-eslint"
  },
  extends: [
    "plugin:vue/recommended"
  ],
  rules: {
    "vue/key-spacing": "error",
    "vue/keyword-spacing": ["error", { before: true, after: true }],
  }
}

```

**What did you do?**	

```vue
<template>
  <div>
    <a @click="doSomething">Click me</a>
  </div>
</template>

<script>
export default {
  name: 'MyComponent',
  methods: {
    doSomething(){
      if(true){return ''}else{return ''}
    },
  },
}
</script>
```

**What did you expect to happen?**

Linter should return vue/keyword-spacing errors

**What actually happened?**

Linter finished without showing errors

Any help greatly appreciated. Thanks!

**Comment by mysticatea** [comment]
Thank you for this issue.

This is intentional behavior.
`vue/keyword-spacing` does not replace core `keyword-spacing` rule. Use the core `keyword-spacing` rule for scripts in the same manner until now.
